Upstream patch from 41.1 release

From 42b0362e9844f4c5114b21464185620eaadcfcaa Mon Sep 17 00:00:00 2001
From: Albert Vaca Cintora <albertvaka@gmail.com>
Date: Mon, 25 Oct 2021 00:07:52 +0200
Subject: [PATCH] build: Remove incorrect i18n.merge_file argument

The positional argument was being silently ignored until meson 0.60.0 where
it fails with "ERROR: Function does not take positional arguments".

See: https://github.com/mesonbuild/meson/issues/9441
---
 data/meson.build | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index f27426a5c..913ffd7b8 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -20,7 +20,6 @@ desktop_conf = configuration_data()
 desktop_conf.set('icon', application_id)
 
 desktop = i18n.merge_file(
-  'desktop',
   input: configure_file(
     input: files('org.gnome.Nautilus.desktop.in.in'),
     output: 'org.gnome.Nautilus.desktop.in',
@@ -34,7 +33,6 @@ desktop = i18n.merge_file(
 )
 
 desktop_autorun_software = i18n.merge_file(
-  'desktop-autorun-software',
   input: 'nautilus-autorun-software.desktop.in',
   output: 'nautilus-autorun-software.desktop',
   install: true,
@@ -48,7 +46,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('appid', application_id)
 
 appdata = i18n.merge_file(
-  'appdata',
   input: configure_file(
     input: files('org.gnome.Nautilus.appdata.xml.in.in'),
     output: 'org.gnome.Nautilus.appdata.xml.in',
-- 
GitLab

